### PR TITLE
DM-47068: Remove nightlyStep5, since it's all faro tasks being dropped.

### PR DESCRIPTION
**Pull request by TallJimbo**

### CODE REVIEW

File: bin/measureHscRC2Metrics.sh
Comment (leeskelvin) [inline bin/measureHscRC2Metrics.sh:1]: The edits here are fine, although do we also want to consider removing the `faro` calls on the lines below?

With that said, as mentioned on the `drp_pipe` PR, this script has been entirely superceded by `run_rc2_subset.sh`, and should probably be retired. I wouldn't object to this entire file being removed on this ticket, if you feel it's appropriate.

**Review by leeskelvin — APPROVED** [review]

File: bin/measureHscRC2Metrics.sh
Comment (TallJimbo) [reply bin/measureHscRC2Metrics.sh:1]: I've gone ahead and deleted the script.

Comment (erykoff) [reply bin/measureHscRC2Metrics.sh:1]: Turns out that script is what jenkins runs.

Comment (leeskelvin) [reply bin/measureHscRC2Metrics.sh:1]: Being fixed on [DM-47137](https://rubinobs.atlassian.net/browse/DM-47137), currently undergoing testing.

**Pull request merged**